app/controllers/catalog_controller.rb

```diff
@@ -1422,7 +1422,7 @@ def rearrange_groups_array
 
   def get_available_resources(kls)
     @edit[:new][:available_resources] = {}
-    kls.constantize.all.each do |r|
+    kls.constantize.all.reject { |st| st.type == "ServiceTemplateAnsiblePlaybook" }.each do |r|
```

Comment:
Better to query for only the items you want:
`kls.constantize.where("type is null or type != 'ServiceTemplateAnsiblePlaybook'").each do |r|`

cc @syncrou